qcoptions.py

```diff
@@ -8,7 +8,9 @@
                   help="Minimum value of scan parameter", metavar="scanmin", default=0)
 parser.add_option("--scanmax", type="int", dest="scanmax",
                   help="Maximum value of scan parameter", metavar="scanmax", default=254)
-parser.add_option("--vfatmask", type="int", dest="vfatmask", 
-                  help="VFATs to be masked in scan & analysis applications (e.g. 0xFFFFF masks all VFATs)", metavar="vfatmask", default=0x0)
+parser.add_option("--vfatmask", type="int", dest="vfatmask",
+		              help="VFATs to be masked in scan & analysis applications (e.g. 0xFFFFF masks all VFATs)", metavar="vfatmask", default=0x0)
+parser.add_option("--stepSize", type="int", dest="stepSize", 
```

Comment:
Right now are run_scans.py and ultraLatency.py the only ones that use the `stepSize` option?  I think yes, please keep `stepSize` option hard coded in only these two scripts to avoid confusion.  For example if you call:

`ultraThreshold.py -h`

You'll see the `stepSize` option right now but it doesn't do anything in `ultraThreshold.py.`

Please update,

Also, keep the options menu's alphabetized for readability.